lighthouse-core/runner.js

```diff
@@ -169,6 +169,16 @@ class Runner {
         assetSaver.saveLhr(lhr, path);
       }
 
+      // Build report if in local dev env so we don't have to run a watch command.
```

Comment:
Is this really necessary? Iterating on the report is rare enough to make this wasted work most of the time, and I think this is going to lead to all sorts of unexpected behavior in the wild (for example this would trigger a build of the report in many LH web projects today because the `cwd` is a git repo and `dist` is the output directory for the webapp build files).

What is your primary opposition to a watch command?

Reply:
> What is your primary opposition to a watch command?

It's another thing to remember, and a "gotcha" for new contributors. Would checking ``__dirname` + '...'` be sufficient?

Reply:
> Would checking ``__dirname + '...' be sufficient?

Not sufficient enough, IMO, to risk a fatal UX experience in exchange for a minor improvement to DX. 

> It's another thing to remember, and a "gotcha" for new contributors

OK, I can empathize with this. How about modifying `yarn start` to build the report first?

Reply:
> What is your primary opposition to a watch command?

I think it is worth getting a larger consensus about this and discussing alternatives. Remembering to do a rebuild or run watch is annoying (I've recently run into forgetting to rebuild e.g. devtools and taking a while to figure out why CI and local tests are disagreeing, for instance). And iterating on the report is rare...unless you're working on report stuff or reviewing others' report PRs.

We've also played this game in Lighthouse before way back when we used handlebars, and it was a relief to all (I thought :) when we got rid of it.

Reply:
yarn start seems fine

Reply:
OK, write it down somewhere then. I'm sure there's soon-to-be broken references that need to be fixed as well.

Reply:
It's in the other PR, we can talk about it there?